internal/view/browser.go

```diff
@@ -585,12 +585,17 @@ func (b *Browser) namespaceActions(aa *ui.KeyActions) {
 	aa.Add(ui.Key0, ui.NewKeyAction(client.NamespaceAll, b.switchNamespaceCmd, true))
 	b.namespaces[0] = client.NamespaceAll
 	index := 1
-	for _, ns := range b.app.Config.FavNamespaces() {
+	favNamespaces := b.app.Config.FavNamespaces()
+	for _, ns := range favNamespaces {
 		if ns == client.NamespaceAll {
 			continue
 		}
-		aa.Add(ui.NumKeys[index], ui.NewKeyAction(ns, b.switchNamespaceCmd, true))
-		b.namespaces[index] = ns
+		if numKey, ok := ui.NumKeys[index]; ok {
+			aa.Add(numKey, ui.NewKeyAction(ns, b.switchNamespaceCmd, true))
+			b.namespaces[index] = ns
+		} else {
+			log.Warn().Msgf("No number key available for favorite namespace %s (%d of %d). Skipping...", ns, index, len(favNamespaces))
```

Comment:
I think we should just break out here. Once there are no more buckets left, no point of keeping the iteration going.
Also the index should only be incremented when we find a slot.

Reply:
Thanks for the feedback, should be fixed now.